ytmusicapi/parsers/browsing.py

```diff
@@ -288,3 +287,17 @@ def parse_related_artist(data):
         'subscribers': subscribers,
         'thumbnails': nav(data, THUMBNAIL_RENDERER),
     }
+
+
+def parse_tasteprofiles(data):
```

Comment:
Not sure this needs to be in parsers/browsing, since it's not being reused anywhere (and likely won't be in the future). Unless you can think of some case where i might be reused, of course.